gravitee-node-api/src/main/java/io/gravitee/node/api/secrets/runtime/discovery/Definition.java

```diff
@@ -0,0 +1,21 @@
+package io.gravitee.node.api.secrets.runtime.discovery;
```

Comment:
I am honestly wondering if this shouldn't be somewhere else as it is implemented only by a APIM plugin not a node plugin.

Reply:
As discussed, this can be used by AM and APIM and the plugin, hence it is here...